env.example

```diff
@@ -14,6 +14,9 @@ HTTPS_PORT=8443
 # System time zone.
 TZ=Europe/Amsterdam
 
+# Public domain name of the web service.
+PUBLIC_DOMAIN="meet.example.com"
```

Comment:
please leave it commented out and move it to the advanced section, also I think we can remove the quotes